sdk/communication/Azure.Communication.Rooms/src/Models/CommunicationRoom.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Core;
+
+namespace Azure.Communication.Rooms
+{
+    /// <summary> The meeting room. </summary>
+    public partial class CommunicationRoom
```

Comment:
That is the plan in upcoming refresh Ted.
Background: 
On server side the mapping is called RoomModel. On the SDK side to have a better name instead of Model tag
we added the custom class CommunicationRoom.
Plan is to update the Service and SDK in upcoming revisions post public preview.

Reply:
Could you get away with https://github.com/azure/autorest.csharp#rename-a-model-class instead of copying the whole implementation?  It doesn't have to happen for this check in, but it might save you time in the long run.

Reply:
Good reference. Thanks Ted. Will address during upcoming refresh.